README.md

```diff
@@ -16,4 +16,19 @@ Just click here to start:
 
 ## Python
 
-Tutorials in [python](https://www.python.org) programming language for [reforms](http://openfisca.org/doc/reforms.html) creation on the French tax and benefit system.
+### [Python tutorials for OpenFisca-France](./python/)
+
+With the Python [Tutorials](./python/) (in French), you will learn how to:
+- use the python API to run calculations
+- get results on the current legislation as well as on reforms
+- export calculation result to CSV
+
+### [Situations for OpenFisca-France](./python/situations)
+This directory contains pre-filled situations extracted from the French "Livret du pouvoir d'achat".
+
+You can use them 'as is' in the web API or import them in the python API as shown in the python tutorials. 
```

Comment:
> Ajouter le lien vers https://legislation.openfisca.fr/swagger dans `web API` ?

Reply:
I would like to not have too many external links because I saw a user click on links in the tutorial read me thinking it was links to tutorials.